Can't clone Terraform modules via `git+ssh` protocol when using the Docker image

Given a Terraform configuration where a module is referenced as follows:
```hcl
module "example" {
  source = "git@github.com:<org>/<repo>.git//modules/example"
}
```

when running the latest [Checkov GitHub Action](https://github.com/bridgecrewio/checkov-action), Checkov fails to download the module and therefore fails to proceed with any subsequent checks.
```
checkov -d infrastructure/production    --quiet   --output github_failed_only --soft-fail --download-external-modules true      
Error: -16 16:31:34,643 [MainThread  ] [ERROR]  failed to get git::ssh://git@github.com/<org>/<repo> because of Cmd('git') failed due to: exit code(128)
  cmdline: git clone -v --depth=1 ssh://git@github.com/<org>/<repo> /github/workspace/infrastructure/production/.external_modules/git@github.com/<org>/<repo>/HEAD
  stderr: 'Cloning into '/github/workspace/infrastructure/production/.external_modules/git@github.com/<org>/<repo>/HEAD'...
error: cannot run ssh: No such file or directory
fatal: unable to fork
```
This is caused by the container image (which the action uses) simply not having the ssh client.

--- a/Dockerfile
+++ b/Dockerfile
@@ -10,6 +10,7 @@ RUN set -eux; \
             git \
             wget \
             curl \
+            openssh-client \
     ; \
     \
     wget -q -O get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3; \

